doc/api/buffer.md

```diff
@@ -872,15 +877,17 @@ console.log(buf.toString('ascii'));
 added: v0.11.13
 -->
 
-* `target` {Buffer}
+* `target` {Buffer} A `Buffer` to compare to
 * `targetStart` {Integer} The offset within `target` at which to begin
-  comparison. default = `0`.
-* `targetEnd` {Integer} The offset with `target` at which to end comparison.
-  Ignored when `targetStart` is `undefined`. default = `target.byteLength`.
+  comparison. **Default:** `0`
+* `targetEnd` {Integer} The offset with `target` at which to end comparison
+  (not inclusive). Ignored when `targetStart` is `undefined`.
```

Comment:
tbh, I’d prefer `exclusive` over `not inclusive` here, but that’s certainly a question of personal taste.